apps/omg/mix.exs

```diff
@@ -32,7 +32,7 @@ defmodule OMG.MixProject do
     [
       {:ex_rlp, "~> 0.5.2"},
       {:merkle_tree, "~> 1.6"},
-      {:deferred_config, "~> 0.1.1"},
+      {:phoenix_pubsub, "~> 1.0"},
```

Comment:
Just out of curiosity, why was this necessary? I didn't notice any changes to how we're using `Phoenix.PubSub`

Reply:
Ah yes, indeed I forgot to remove it!